velox/experimental/wave/common/tests/CudaTest.cpp

```diff
@@ -1635,7 +1640,7 @@ TEST_F(CudaTest, roundtripMatrix) {
 }
 
 TEST_F(CudaTest, addRandom) {
-  constexpr int32_t kNumInts = 16 << 20;
+  int32_t kNumInts = FLAGS_num_random_ints;
```

Comment:
Can you rename it to `numInts` since it's no longer a compile time constant?

Reply:
Done